docs/en/ufs/S3.md

```diff
@@ -162,36 +162,6 @@ alluxio.underfs.s3.proxy.port=<PROXY_PORT>
 
 `<PROXY_HOST>` and `<PROXY_PORT>` should be replaced the host and port for your proxy.
 
-### Configuring Application Dependency
```

Comment:
totally irrelevant for this doc. I Don't understand why it was put here initally.